packages/messageport/tizen/src/messageport_tizen_plugin.cc

```diff
@@ -206,36 +198,27 @@ class MessageportTizenPlugin : public flutter::Plugin {
 
```

Comment:
The first argument of `result->Error` should be an error code, something like "Invalid arguments".